extras/createdevproject.py

```diff
@@ -0,0 +1,47 @@
+"""
+Modifies the project template to fit with local development
+"""
+import os
```

Comment:
There should be a blank line separating module-level docstring from first import.